<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="slotTab">
       <attribute name="title">
        <string>スロット</string>
       </attribute>
       <layout class="QHBoxLayout" name="slotTabLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="settingWidget">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>チャット接頭辞：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="chatPrefixEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>スロット名：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="slotComboBox"/>
              </item>
              <item>
               <widget class="QPushButton" name="editSlotListButton">
                <property name="text">
                 <string>スロット一覧を編集する</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="saveLogCheckBox">
                <property name="text">
                 <string>スロットログをファイルに記録する</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>記録先：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="logDirEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="dirSelectButton">
                <property name="text">
                 <string>参照</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;記録先フォルダには以下のファイル名で関連ログが保存されます。&lt;/p&gt;&lt;p&gt;スロットログ：(スロット名)_log_(日時).log&lt;/p&gt;&lt;p&gt;スロット統計情報 (結果)：(スロット名)_info_(日時).txt&lt;/p&gt;&lt;p&gt;また、統計タブでは記録先フォルダから該当スロット名の統計情報ファイルを取得して集計結果を算出しています。&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;(注意) 統計情報ファイルは「終了」ボタンを押して正常に終了しないと記録されません！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::MarkdownText</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>ログファイル：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="pathEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="fileSelectButton">
                <property name="text">
                 <string>参照</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="startButton">
              <property name="text">
               <string>開始</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="mainWidget">
           <layout class="QHBoxLayout" name="mainLayout">
            <item>
             <widget class="QWidget" name="logWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QTextEdit" name="textLogView">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="swapButton">
                   <property name="text">
                    <string>左右入れ替え</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pauseButton">
                   <property name="text">
                    <string>停止</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="stopButton">
                   <property name="text">
                    <string>終了</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="historyTab">
       <attribute name="title">
        <string>統計</string>
       </attribute>
       <layout class="QVBoxLayout" name="historyLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>スロット名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="historySlotEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="historyLoadButton">
          <property name="text">
           <string>実行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
